scss/input/_layout.scss

```diff
@@ -5,38 +5,47 @@
     $textbox-shadow: 0 2px 2px 1px rgba(0, 0, 0, .06) !default;
     $fieldset-margin: 30px;
 
-    .k-textbox > input,
-    .k-input,
-    .k-textbox {
+    .k-textbox,
+    .k-maskedtextbox {
+        @include border-radius( $border-radius );
+        padding: $input-padding-y $input-padding-x;
+        width: 12.4em;
```

Comment:
The width is somewhat hardcoded in https://github.com/telerik/kendo-theme-default/blob/master/scss/common/_forms.scss. I've used the same width.

I was also wondering if I should put `k-maskedtextbox` _forms.